src/support.js

```diff
@@ -510,7 +510,11 @@ function loadWebAssemblyModule(binary, flags) {
           assert(parts.length == 3)
           var name = parts[1];
           var sig = parts[2];
+#if WASM_BIGINT
+          var legalized = false;
+#else
           var legalized = sig.indexOf('j') >= 0; // check for i64s
```

Comment:
Maybe `resolveSymbols` code should explictly ignore `legalized` arg (ifdef out all the code there that handles it and assert its not true).

Reply:
I considered that, but it's a bunch more ifdefing and it makes the code less clear, I think?

Reply:
Rewrote it like that way, as I don't have a strong preference.